lkp-exec/install

```diff
@@ -227,7 +227,7 @@ install_packages() {
 	fi
 	
 	if [ "$distro" = "exaleap-riscv-linux" ]; then
-	echo "riscv64"
+		echo "riscv64"
```

Comment:
thanks, this commit need to be amended with earlier one, so the whole patchset is clean without intermediate issue. Thus pls do a rebase for the pr.

Reply:
> thanks, this commit need to be amended with earlier one, so the whole patchset is clean without intermediate issue. Thus pls do a rebase for the pr.

Will do & update.

Reply:
> thanks, this commit need to be amended with earlier one, so the whole patchset is clean without intermediate issue. Thus pls do a rebase for the pr.

The pr has been rebased, please review.